components/src/status_im/ui/components/react.cljs

```diff
@@ -36,6 +36,7 @@
 (def net-info (get-react-property "NetInfo"))
 (def view (get-class "View"))
 (def safe-area-view (get-class "SafeAreaView"))
+(def progress-bar (get-class "ProgressBarAndroid"))
```

Comment:
i'm not sure about using this progress bar, it may be confusing, here https://github.com/status-im/status-react/issues/4345 we implemented status loading bar for both platforms, maybe better to use it instead

Reply:
@flexsurfer we only need to support Android platform and this progress bar is already built-in into RN so no extra dependency.
I don't understand how to use progress bar from #4345, could you me point me to how to use it? I only found this https://github.com/status-im/status-react/blob/f8978ca0aa38a8b7f5ba000337bdebae976e9a9b/src/status_im/ui/screens/browser/views.cljs#L107
Also, where can i see how it looks.

Reply:
Agree it's worth considering the existing progress bar

Reply:
@jeluard i'm happy to use it, but i don't see easy way how to use for my case. Do you guys mean i could somehow use `loading-indicatior` fn which i linked in previous comment?